documentation/docs/steps/transportRequestUploadCTS.md

```diff
@@ -0,0 +1,173 @@
+* You have the [S_DEVELOP authorization](https://sapui5.hana.ondemand.com/#/topic/a883327a82ef4cc792f3c1e7b7a48de8.html) for operations on a SAPUI5 ABAP repository.
+* You have created a transport request on the ABAP system, which is the target container of the SAPUI5 application to upload.
+
+## Upload Client Setup
+
+The step `transportRequestUploadCTS` uses the [Node.js](https://nodejs.org)-based [SAP Fiori tools](https://help.sap.com/viewer/product/SAP_FIORI_tools/Latest/en-US) to upload your SAPUI5 application to the UI5 repository service of your ABAP back-end infrastructure. It performs a deployment running the [Fiori deploy command](https://www.npmjs.com/package/@sap/ux-ui5-tooling#fiori-deploy---performs-the-deployment-of-the-application-into-an-abap-system) on a Docker image.
+By default, a plain [Node.js Docker image](https://hub.docker.com/_/node) is pulled and equipped with the SAPUI5 toolset during runtime of the pipeline.
+Alternatively, you can provide your own, fully equipped Docker image. This speeds-up the upload process, but obligates to maintain and provision the image on a Docker registry.
+
+### Fully Equipped Docker image
+
+To create an own Docker image with the [SAP Fiori tools](https://help.sap.com/viewer/product/SAP_FIORI_tools/Latest/en-US) installed, proceed as follows:
+
+1. Create a Node-based Docker image with the SAP Fiori tools installed:
+
+   ```Dockerfile
+   FROM node
+   USER root
+   RUN npm install -global @ui5/cli @sap/ux-ui5-tooling @ui5/logger @ui5/fs
+   USER node
+   ```
+
+   ```/bin/bash
+   docker build -t my/fiori-node .
+   ```
+
+1. Push your image to your private [Docker Hub registry](https://hub.docker.com/).
+
+   ```/bin/bash
+   docker push my/fiori-node  
+   ```
+
+1. Add the following to your config.yml file:
+
+```yaml
+steps:
+  transportRequestUploadCTS:
+    dockerImage: 'my/fiori-node'
+    deployToolDependencies: []
+```
+
+## Building the SAPUI5 Application
+
+Build your SAPUI5 application with the build command of the SAPUI5 toolset and use the step [npmExecuteScripts](npmExecuteScripts.md) to run the build command, proceeding as follows:
+
+1. Configure the step in the `package.json` file of your project
+
+   ```json
+   {
+      ...
+      "scripts": {
+         "start": "ui5 serve",
+         "test": "npm run lint",
+         "build": "ui5 build --clean-dest",
+         ...
+      },
+      "dependencies": {},
+      "devDependencies": {
+         "@ui5/cli": "^2.11.2",
+         ...
+      }
+   }
+   ```
+
+1. Setup the execution step in the pipeline
+
+   ```groovy
+   stage('Build') {
+      npmExecuteScripts(script: this, runScripts: ['build'])
+   }
+   ```
+
+**Note:** Do not use the `mtaBuild` step. The `mta` is dedicated to the SAP Cloud Platform. It does neither create the expected `dist` folder nor the compliant content.
```

Comment:
Do not use the `mtaBuild` script*. The MTA Build Tool* is dedicated to the SAP Cloud Platform. It does neither create the expected `dist` folder nor the compliant content.

Reply:
I'll keep the **Note:**.

Reply:
Yes, I forgot to add it at the beginning :)